tests/acceptance/clinical_browser.js

```diff
@@ -182,6 +188,42 @@ function clinVarSearchByGene(driver){
 
     return driver;
 }
+
+function emtpyPositionFilter(driver){
```

Comment:
This method should be split in 3 separate test cases. Also `empty` instead of `emtpy`.